components/gitpod-db/package.json

```diff
@@ -4,8 +4,10 @@
   "version": "0.1.5",
   "scripts": {
     "prepare": "yarn clean && yarn build",
-    "build": "npx tsc",
+    "build": "yarn lint && npx tsc",
     "build:clean": "yarn clean && yarn build",
+    "lint": "yarn eslint src/**/*.ts",
+    "format": "yarn eslint src/**/*.ts --fix",
     "rebuild": "yarn build:clean",
     "build:watch": "watch 'yarn build' .",
     "test": "yarn db-test",
```

Comment:
Same for `watch` here: Should call `yarn build` so we see the lint errors during dev.

Reply:
Are you using watch? I do never (maybe I should)

Reply:
Using it exclusively, as it builds all (transient) deps as well. :+1:

Reply:
> Should call yarn build so we see the lint errors during dev.

looks to me like it does

Reply:
`tsc -w --preserveWatchOutput`. Or is tsc calling eslint? :bulb: (already stopped my workspace..)